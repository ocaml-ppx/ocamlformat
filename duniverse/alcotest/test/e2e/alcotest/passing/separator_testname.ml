let () =
  Alcotest.run __FILE__
    [
      ( "with/separator",
        [ Alcotest.test_case "First test case" `Quick (fun () -> ()) ] );
    ]
