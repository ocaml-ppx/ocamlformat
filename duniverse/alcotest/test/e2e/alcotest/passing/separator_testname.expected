Testing `test/e2e/alcotest/passing/separator_testname.ml'.
This run has ID `<uuid>'.

  [OK]          with/separator          0   First test case.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 1 test run.
